examples/event-processor/demo-bulk-posting.sh

```
@@ -0,0 +1,78 @@
+
+set -e
+
+echo "=============================================="
+echo "ChronoQueue Bulk Posting Demo"
+echo "=============================================="
+echo ""
+
+# Colors for output
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+YELLOW='\033[1;33m'
```

Shellcheck (0.11.0)

[warning] YELLOW appears unused. Verify use (or export if used externally).

(SC2034)